source/guides/references/protractor-migration-guide.md

```diff
@@ -0,0 +1,565 @@
+---
+title: Protractor Migration Guide
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- Benefits of using Cypress in Angular apps
+- How Cypress can create reliable e2e tests for Angular apps
+- How to migrate Protractor tests into Cypress
+{% endnote %}
+
+
+## Introduction
```

Comment:
Main headings should start with h1 `#`, see main note about nesting headings suggestion.

```suggestion
# Introduction
```

Reply:
Got it. In the future, we may want to look into restructuring how the sidebar gets generated since this leads to multiple h1's on the page which makes it difficult for screen readers and accessibility content-wise.